### PR TITLE
Remove private method

### PR DESCRIPTION
**Pull request by canyon289**
Also name all arviz imports relative imports

This one was only was mostly due to my OCD. I hope you guys don't judge me too much

### CODE REVIEW

File: arviz/plots/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from .densityplot import plot_density
 from .energyplot import plot_energy
 from .forestplot import plot_forest
-from .kdeplot import plot_kde, _fast_kde, _fast_kde_2d
```
Comment (aloctavodia) [inline arviz/plots/__init__.py]: I introduced this on purpose #235. Sometimes is useful to have access to `_fast_kde` and `_fast_kde_2d`. Although they should be of no interest to most users so I kept the underscore.

Comment (canyon289) [reply arviz/plots/__init__.py]: Ah I see, my apologies. I'll undo this change

Comment (aloctavodia) [reply arviz/plots/__init__.py]: Thanks! Don't worry, lot of changes recently and is hard to follow all of them.

**Comment by canyon289** [comment]
Made the updates
